### DISCUSSION TITLE
Tp - Utilisation fonction

### DISCUSSION THREAD

**Discussion by julienChemillier** [mp2i-fsm/mp2i-2021#11] [Question] [answered]
Bonjour,
Peut-on utiliser la fonction 'atan' dans le TP1 pour la fonction polaire ?
Bien à vous,
Julien

**Comment by fortierq** [comment, marked as answer]
Bonjour,
Oui (atan, acos, asin existent en OCaml, et calculent la réciproque de tan, cos, sin).
Mais mon idée était plutôt de faire la transformation inverse : (r, theta) -> (r cos theta, r sin theta)
Faites comme vous préférez.
